features/karaf/src/main/feature/feature.xml

```diff
@@ -12,8 +12,8 @@
 <features name="${project.artifactId}-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.0.0">
 
     <feature name="openhab-runtime-base" description="openHAB Runtime Base" version="${project.version}">
-        <feature>shk-esh-base</feature>
-        <feature>shk-esh-io-console-karaf</feature>
+        <feature>esh-base</feature>
```

Comment:
> Those have been added to the features where needed.

That is good and correct for the direct dependencies .

Hm, I could be wrong, but there is IMHO stuff that no one depend but should be part of the base runtime (e.g. storage-mapdb).
I would also assume that all the model stuff are seen as part of the openhab-runtime-base feature and so installed every time (regardless if there is something that needs it).